### ISSUE TITLE
[Deque Analysis] Table subheadings lack color contrast (against a non-white background)

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1160]
## Deque Analysis Summary

Table subheadings lack color contrast:

"Element has insufficient color contrast of 3.77 (foreground color: `#777777`, background color: `#e9ecef`, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1"

This was noticed on the "Edit Item" -> "Bitstreams" tab.  This issue occurs in the "BUNDLE: [name]" subheaders within the Bitstreams table.

Deque issue ticket IDs: 470734, 470735, 470736, 470737

## Recommended Fix

Likely make the heading text darker.  If we wanted to keep it gray, we'd need the foreground color (text) to be `#666666` on a background of `#e9ecef`.  I found this using this tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

## More Information / Tools

* As necessary, see mentioned Issue IDs in our assessment results (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by tdonohue** [comment]
@artlowel : assigning this to you for analysis.  This is likely a quick fix

**Comment by artlowel** [comment]
This will be fixed if we put the default bootstrap gray's back, as the proposed for #1154

**Comment by tdonohue** [comment]
Sounds good, marking as a zero hour estimate then since it'll be fixed by #1154
